.gitlab-ci.yml

```diff
@@ -225,8 +225,7 @@ deploy-prod-canary:
   stage: deploy:canary
   extends:
     - .base-configuration
-    - .tags
-  when: manual
```

Comment:
we should not `BUILD_MODE=release yarn build:bundle` for canary anymore otherwise we won't be able to differentiate the commits

Reply:
I created a `canary` build mode, which is the same as `staging`. I found it clearer instead of reusing staging. Wdyt?